gpt_index/data_structs/data_structs.py

```diff
@@ -368,6 +368,15 @@ def get_type(cls) -> str:
         return IndexStructType.QDRANT
 
 
+class MilvusIndexDict(IndexDict):
+    """Index dict for Milvus vector index."""
+
+    @classmethod
+    def get_type(cls) -> str:
+        """Get type."""
+        return IndexStructType.MILVUS
+
+
```

Comment:
nit: this is a legacy file, which we preserved to allow migration of old serialized jsons (via `migrate_v1_to_v2.py` script.

I don't think we need to add the new feature here.

Reply:
removed